src/main/java/com/juansecu/opentoonix/user/UserService.java

```diff
@@ -32,6 +39,9 @@
 import com.juansecu.opentoonix.user.models.entities.UserEntity;
 import com.juansecu.opentoonix.user.models.views.UserInformationView;
 import com.juansecu.opentoonix.usernameblacklist.UsernameBlacklistService;
+import com.juansecu.opentoonix.mail.OnRegistrationCompleteEvent;
```

Comment:
Be sure to import modules alphabetically. If you're using an IDE like IntelliJ, it makes that for you, but make sure to separate the application modules from the Java, Javax or third-party ones.